examples-of-custom-resources/egress-mtls/README.md

```diff
@@ -0,0 +1,60 @@
+# Egress MTLS
+
+In this example, we deploy a web application, configure load balancing for it via a VirtualServer, and apply an Egress MTLS policy.
+
+## Prerequisites
+
+1. Follow the [installation](https://docs.nginx.com/nginx-ingress-controller/installation/installation-with-manifests/) instructions to deploy the Ingress Controller.
+1. Save the public IP address of the Ingress Controller into a shell variable:
+    ```
+    $ IC_IP=XXX.YYY.ZZZ.III
+    ```
+1. Save the HTTP port of the Ingress Controller into a shell variable:
+    ```
+    $ IC_HTTP_PORT=<port number>
+    ```
+
+## Step 1 - Deploy a Secure Web Application
```

Comment:
could we add an explanatory paragraph here. Something like.

The application requires clients to use TLS and present a client TLS certificate which it will verify.